src/api/app/views/webui2/webui/package/add_file.html.haml

```diff
@@ -0,0 +1,26 @@
+- @pagetitle = 'Add File'
+
+.card.package-add-file
+  = render partial: 'tabs', locals: { project: @project, package: @package }
+  .card-body
+    %h3
+      Add file to #{@project.name}/#{@package.name}
```

Comment:
I believe this should be the same as `@pagetitle`

Reply:
Should it?
I was not aware of that :cry: Do we have this documented somewhere?

Reply:
I will document it in the Bootstrap wiki page.

Reply:
Done here: https://github.com/openSUSE/open-build-service/wiki/How-to-work-in-Bootstrap-(new-UI)#implementation-details

Reply:
Updated!